### PR TITLE
placeholder support

### PR DESCRIPTION
**Pull request by bartes**
fallback to placeholder attr when no value

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -51,3 +51,4 @@ src/*.js.map
 
 # Exclude the bundle demo file
 !docs/bundle.js
+package-lock.json
```
Comment (spirosikmd) [inline .gitignore]: We shouldn't ignore [`package-lock.json`](https://docs.npmjs.com/files/package-locks#using-locked-packages), so everyone can get the exact same dependency tree. This repository uses `yarn`. Ideally we should use only yarn and have both `yarn.lock` and `package-lock.json` files.

File: src/elastic-input.directive.ts [hunk 2]
```diff
@@ -66,8 +66,9 @@ export class ElasticInputDirective implements OnInit {
   }
 
   update () {
-    this.mirror.innerText = this.element.nativeElement.value;
+    let domEl = this.element.nativeElement;
```
Comment (spirosikmd) [inline src/elastic-input.directive.ts]: We can set this to `const` as its value is not modified.

File: .gitignore (hunk 1 above)
Comment (bartes) [reply .gitignore]: aah haven't notice that. dropped that commit

File: src/elastic-input.directive.ts (hunk 2 above)
Comment (bartes) [reply src/elastic-input.directive.ts]: Totally agree. I wanted to stay consistent. I changed let to const in all of the places

**Comment by bartes** [comment]
thanks for the info about the contribution guidelines. will follow that.

**Comment by bartes** [comment]
@spirosikmd - should be done. please verify

**Comment by spirosikmd** [comment]
Great @bartes! Thank you very much for your work 👍

**Comment by spirosikmd** [comment]
It would be nice to [add yourself as contributor](https://github.com/spirosikmd/angular2-elastic-input/blob/master/CONTRIBUTING.md#add-yourself-as-a-contributor) as well!

**Pull request merged**